spec/jekyll_seo_tag_integration_spec.rb

```diff
@@ -296,6 +301,7 @@
         expected = <<-EOS
 <!-- Begin Jekyll SEO tag v#{version} -->
 <title>Foo</title>
+<meta name="generator" content="Jekyll v3.5.2" />
```

Comment:
Definitely make this programmatic or else this will fail when built with any other Jekyll version. I don't believe we lock this so CI will upgrade and this will fail. Use `#{Jekyll::VERSION}` like above.